bot/airtable.py

```diff
@@ -0,0 +1,73 @@
+# imports
+import pandas as pd
```

Comment:
It doesn't look like this is being used. I am going to remove and hopefully it fixes the build